features/facebookImport/src/model/importer.js

```diff
@@ -13,7 +13,7 @@ import UnfollowedPagesImporter from "./importers/pages-unfollowed-importer.js";
 import MessagesImporter from "./importers/messages-importer.js";
 import AdminRecordsImporter from "./importers/admin-records-importer.js";
 import AccountSessionActivitiesImporter from "./importers/account-session-activities-importer.js";
-import NameImporter from "./importers/name-importer.js";
+import PersonalDataImporter from "./importers/personal-data-importer.js";
```

Comment:
Is this intended to be expanded with more non-name personal data' Because for the time being, it's still the name, only split.

Reply:
Thats the point. Name address, email, etc. Will expand later, just wanted this to be able to use it to test rdf